### ISSUE TITLE
Compile error when building "PSig expected"

### ISSUE THREAD

**Issue by CatherineGasnier** [LexiFi/landmarks-viewer#3]
I'm trying to build this project but here's what I get:
```
catg@catg-mbp ~/checkouts/landmarks-viewer (master●)$ dune build --profile release
File "bindings/js_core.ml", line 62, characters 11-16:
62 |   include ([%js] : sig
                ^^^^^
Error: PSig expected
File "default/_unknown_", line 1, characters 0-0:
Error: No rule found for landmarks_viewer.bc.js
```

Here's what's in my opam:
```
catg@catg-mbp ~/checkouts/landmarks-viewer (master●)$ opam list          [ruby-2.6.3p62]
# Packages matching: installed
# Name              # Installed    # Synopsis
base-bigarray       base
base-threads        base
base-unix           base
dune                2.9.1          Fast, portable, and opinionated build system
gen_js_api          1.0.8          pinned to version 1.0.8 at git+file:///Users/catg/chec
landmarks           1.4            A simple profiling library
ocaml               4.12.1         The OCaml compiler (virtual package)
ocaml-compiler-libs v0.12.4        OCaml compiler libraries repackaged
ocaml-config        2              OCaml Switch Configuration
ocaml-variants      4.12.1+options Official release of OCaml 4.12.1
ojs                 1.0.8          Runtime Library for gen_js_api generated libraries
ppx_derivers        1.2.1          Shared [@@deriving] plugin registry
ppxlib              0.23.0         Standard library for ppx rewriters
sexplib0            v0.14.0        Library containing the definition of S-expressions and
stdlib-shims        0.3.0          Backport some of the new stdlib features to older comp
```

Any idea?

**Comment by Armael** [comment]
I can reproduce this as well, even after pinning `ojs` and `gen_js_api` to their respective dev repos.

**Comment by jboillot** [comment]
Hello!
I guess you are no longer affected by this issue after so long, but for future visitors, here is a hotfix to make the viewer work https://github.com/jboillot/landmarks-viewer/tree/hotfix/make-it-work :)

**Comment by mlasson** [comment]
Fixed by #4
